### ISSUE TITLE
ee_initialize subscript out of bounds

### ISSUE THREAD

**Issue by ctlamb** [r-spatial/rgee#62]
Hello, I followed the install instructions, but am having trouble with ee_initialize(). And the issue is the same whether I add my email directly, or if I go through the manual log in and copy the code route

`Well done! rgee was successfully set up in your system.
rgee needs restart R to see changes.
Do you want to continues? 

1: yes
2: no

Selection: 1

Restarting R session...

> ee_Initialize(email="lamb.eco.research@gmail.com")
── rgee 0.6.1 ──────────────────────────────────────────────────────────────────────── earthengine-api 0.1.225 ── 
 ✓ email: lamb.eco.research@gmail.com 
 ✓ Initializing Google Earth Engine:  DONE!
Error in ee$data$getAssetRoots()[[1]] : subscript out of bounds`

**Comment by csaybar** [comment]
Hi @ctlamb! 

It seems the error appears in the last mile, could you show me the result of run this?

```r
library(rgee)
ee$Initialize()
ee$data$getAssetRoots()
```

**Comment by ctlamb** [comment]
Thanks! That produces the following:

`> library(rgee)

Attaching package: ‘rgee’

The following object is masked _by_ ‘.GlobalEnv’:

    ee

> ee$Initialize()
Error: attempt to apply non-function
> ee$data$getAssetRoots()
Error: attempt to apply non-function`

**Comment by csaybar** [comment]
It seems that you assign to ee another variable. ee is a reserved word in rgee. Please be sure to remove other assignments to "ee".

```r
library(rgee)
ee <- emptyenv()
ee$Initialize()
# attempt to apply non-function

rm(ee)
ee$Initialize()
ee$data$getAssetRoots()
```

**Comment by ctlamb** [comment]
Ooops my mistake. Here it is with the ee variable fixed.

```
> rm(ee)
> ee$Initialize()
> ee$data$getAssetRoots()
list()
```

**Comment by csaybar** [comment]
I committed a mistake in the code, I should've used "error" rather than "finally". Sorry

https://github.com/r-spatial/rgee/blob/2cbe73fd2e0ade120276700a0a2584171f1df60d/R/ee_Initialize.R#L231

`rgee` assumes that users have an Earth Engine Assets folder. And in your case, you haven't created one yet. You can create your EE assets folder running:

`ee$data$createAssetHome("users/PUT_YOUR_NAME_HERE")`

But take care because once created you will not be able to change the folder name again. If the root folder was created successfully, you will execute  ee_Initialize() without problems!.

**Comment by ctlamb** [comment]
This worked perfect. I am up and running. Thank you !

Best,
Clayton

**Comment by csaybar** [comment]
We solved the problem in rgee 0.6.2. I going to close the issue but feel free to reopen if you consider is necessary!.
